man/systemd.exec.xml

```diff
@@ -1899,6 +1901,8 @@ SystemCallErrorNumber=EPERM</programlisting>
         <para>Variables set for invoked processes due to this setting are subject to being overridden by those
         configured with <varname>Environment=</varname> or <varname>EnvironmentFile=</varname>.</para>
 
+        <para>Since v239 systemd supports newlines in enviroment variables passed through <varname>PassEnvironment=</varname></para>
```

Comment:
Instead of just talking about newlines, the text should be changed to say that c-escapes are allowed in the text, and mention that this can be used to insert newlines, tabs, and such.

Right now, from this description it is not clear if newlines can be inserted verbatim or escaped.

Reply:
As a side note, the documentation usually doesn't mention systemd versions. In case like these, the file NEWS is modified instead...